plugins/catalog-graph/src/components/CatalogGraphPage/SelectedOwnbyFilter.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { alertApiRef, useApi } from '@backstage/core-plugin-api';
+import { catalogApiRef } from '@backstage/plugin-catalog-react';
+import {
+  Box,
+  Checkbox,
+  FormControlLabel,
+  makeStyles,
+  TextField,
+  Typography,
+} from '@material-ui/core';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Autocomplete } from '@material-ui/lab';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import useAsync from 'react-use/lib/useAsync';
+
+const useStyles = makeStyles({
+  formControl: {
+    maxWidth: 300,
+  },
+});
+
+export type Props = {
+  value: string[] | undefined;
+  onChange: (value: string[] | undefined) => void;
+};
+
+export const SelectedOwnByFilter = ({ value, onChange }: Props) => {
+  const classes = useStyles();
+  const alertApi = useApi(alertApiRef);
+  const catalogApi = useApi(catalogApiRef);
+
+  const { error, value: ownedby } = useAsync(async () => {
+    return await catalogApi.getEntityFacets({ facets: ['Group', 'User'] }).then(
+      response =>
+        response.facets.Group.concat(response.facets.User)
+          ?.map(f => f.value)
+          .sort() || [],
+    );
+  });
+
+  useEffect(() => {
+    if (error) {
+      alertApi.post({
+        message: `Failed to load entity kinds`,
+        severity: 'error',
+      });
+    }
+  }, [error, alertApi]);
+
+  const normalizedOwnedby = useMemo(
+    () =>
+      ownedby
+        ? ownedby.map(owner => owner.toLocaleLowerCase('en-US'))
```

Comment:
You should write all of your code to deal with entity refs only. Those are always lowercased and on a normalized form so they are safe to just string compare. Just the name is never safe to use, because there can be both `user:default/foo`, `user:other-namespace/foo`, and `group:third-namespace/foo` all at the same time